src/pages/academy/blog/_all-articles.tsx

```diff
@@ -14,9 +16,24 @@ type AllArticlesProps = {
 export type ItemType = ArticleDataType[0]
 
 const AllArticles = ({ article_data }: AllArticlesProps) => {
+    const [is_mobile] = useBrowserResize()
+    const [current_page, setCurrentPage] = useState(1)
+    const posts_per_page = is_mobile ? 9 : 19
```

Comment:
Can we have the variable for  `desktop_max_pages` & `mobile_max_pages` so later everyone can adjust it easily. Also if we can make it accurate like 10 & 20 and you do the math on your side.